src/main/java/org/eclipse/uprotocol/transport/builder/UMessageBuilder.java

```diff
@@ -71,7 +71,7 @@ public static UMessageBuilder publish(UUri source) {
      */
     public static UMessageBuilder notification(UUri source, UUri sink) {
         Objects.requireNonNull(source, "source cannot be null.");
-        Objects.requireNonNull(sink, "sink cannot be null.");
+        Objects.requireNonNull(sink, "sink cannot be null. for Notification");
```

Comment:
Remove `.` in the error message

Reply:
good catch, I was messing around with that for testing a bug but forgot to remove it, thank you Neelam!